client/src/core/client/admin/components/ModerateCard/MarkersContainer.tsx

```diff
@@ -33,6 +33,13 @@ const markers: Array<
       </Localized>
     )) ||
     null,
+  (c) =>
+    (c.status === "PREMOD" && c.author.premoderatedBecauseOfEmailAt && (
+      <Localized id="moderate-marker-preMod-userEmail" key={keyCounter++}>
```

Comment:
Can you please add this localization? I'm not seeing it in `admin.ftl`

Reply:
fixed!